Tools/autotest/arduplane.py

```diff
@@ -2062,6 +2053,53 @@ def change_speed():
         if ex is not None:
             raise ex
 
+    def test_fence_alt_ceil_floor(self):
+        fence_bit = mavutil.mavlink.MAV_SYS_STATUS_GEOFENCE
+        self.set_parameter("FENCE_TYPE", 9)     # Set fence type to max and min alt
+        self.set_parameter("FENCE_ACTION", 0)   # Set action to report
+        self.set_parameter("FENCE_ALT_MAX", 200)
+        self.set_parameter("FENCE_ALT_MIN", 100)
+
+        # Grab Home Position
+        self.mav.recv_match(type='HOME_POSITION', blocking=True)
+        self.homeloc = self.mav.location()
+
+        cruise_alt = 150
+        self.takeoff(cruise_alt)
+
+        self.do_fence_enable()
+
+        self.progress("Fly above ceiling and check for breach")
+        self.change_altitude(self.homeloc.alt + cruise_alt + 80)
+        m = self.mav.recv_match(type='SYS_STATUS', blocking=True)
+        print("%s" % str(m))
```

Comment:
self.progress

Reply:
Only replying once, given there are several review comments to use `self.progress`.

I disagree this should be `self.progress` - This is not a progress statement, this is to print out the value returned by `self.mav.recv_match(type=....)`. This is also in line with many other test cases already present here.

Some examples of this occurring in multiple test files.:
* `arduplane.py`
  * `test_throttle_failsafe`
  * `test_throttle_failsafe_fence`
  * `test_adsb`
* `arducopter.py`
  * `test_manual_control`

Reply:
@joshanne sorry, @khancyr is correct here.

The other places you've found are.... inadequacies... in the code.  They tend to slip in every now and then.

Reply:
... of course. Updated.